validators/validator.go

```diff
@@ -28,5 +28,8 @@ func New() *validator.Validate {
 	_ = v.RegisterValidation(HTTPStatusCode.Key, HTTPStatusCode.Func)
 	_ = v.RegisterValidation(HTTPStatusCodeRange.Key, HTTPStatusCodeRange.Func)
 
+	// * Default
+	_ = v.RegisterValidation(Default.Key, Default.Func)
```

Comment:
Instead of registering a new validator, it might be worth registering a new tag with the [RegisterTagNameFunc](https://pkg.go.dev/github.com/go-playground/validator/v10@v10.26.0#Validate.RegisterTagNameFunc) function. This allows you to use a `default` tag that influences validation.

Example before:

`validate:"min:10,max:100,default=50"`

Example after:

`default:"50" validate:"min:10,max:100"`